### PR TITLE
Update Example6 and Example7

### LINKED ISSUE #128: Revisit Examples 4 and 5
@marcoscaceres requested that we revisit Examples 4 ( Handling the paymentrequest event)  and 5 (Simple Payment Handler Window). I am recording this issue to do so.

If we can revisit the examples and get some support over the next week, then I am fine to update the examples for the FPWD. Otherwise, I will mark them with this issue marker and we will revisit them after FPWD.

Ian

### LINKED ISSUE #280: window.open not working from paymentrequest eventlistener
window.open does not work from paymentrequest event listener below. Is there any way I can pop up a window on ServiceWorker/Payment Handler for user to select a card from a list of cards on indexedDB after 'Pay' button is clicked from the Merchant website. ServiceWorker stored the list of cards on indexedDB during install.

self.addEventListener('paymentrequest', (evt) => {
    console.log('paymentrequest evt is ' + evt);
    var popup =  window.open("", "newWindow", "width=800, height=600");
});

### PR DESCRIPTION
**Pull request by romandev**
closes #128, 
closes #280 

This change includes the following things:
  - Use async/await
  - Use correct message handler

***
<a href="https://pr-preview.s3.amazonaws.com/romandev/payment-handler/pull/290.html" title="Last updated on Apr 12, 2018, 5:59 PM GMT (ed7c106)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/payment-handler/290/d9ebc35...romandev:ed7c106.html" title="Last updated on Apr 12, 2018, 5:59 PM GMT (ed7c106)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1734,26 +1739,26 @@ <h2>
...
+  const paymentAppResponse = {
+    methodName: "basic-card",
+    details: basicCardResponse
+  };
 
-    e.source.postMessage(paymentAppResponse);
-    window.close();
-  }
+  navigator.serviceWorker.controller.postMessage(paymentAppResponse);
+  window.close();
```
Comment (rsolomakhin) [inline index.html:1761]: `window.close()` will have no effect (at least in Chrome). Let's remove it. Chrome closes the payment handler window when the service worker responds to the `paymentrequest` event.

**Review by rsolomakhin — DISMISSED** [review]
@romandev: One nit. Otherwise looks OK.

File: index.html (hunk 1 above)
Comment (romandev) [reply index.html:1761]: If you don't mind, I'll keep this PR for now until https://github.com/w3c/payment-handler/issues/300 issue is resolved.
A few month ago, I left a review comment in CL: https://chromium-review.googlesource.com/c/chromium/src/+/1019941#message-063bd9d764584bf84445f34f2afb435a25b5d8db
I don't remember exactly but I think closing opened window by `respondWith()`'s promise seems a little bit strange to me.

**Review by marcoscaceres — APPROVED** [review]
Going to go with this as I'm doing some editorial fixes. We can deal with the `window.close()` nit separately.

**Pull request merged**